src/factories/plugin-factory.ts

```diff
@@ -81,6 +83,13 @@ const pluginFactories: Record<string, PluginBuilder> = {
       options.repositoryConfig,
       (options.type as SentenceCasePluginConfig).specialWords
     ),
+  'group-priority': options =>
```

Comment:
Eventual cleanup, it would be nice if we could figure out how t instantiate plugins without needing to enumerate them all in this lookup. Perhaps we could introduce a generated `index.ts` to the plugins folder, which would walk the plugins folder during a build step.